eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -32,59 +42,94 @@ function Get-SwaggerInfo()
         Write-Error $_
     }
     Write-Host "Cannot find swagger info"
-    Pop-Location
     exit 1
 }
 
-function Update-AutorestConfigFile() {
+function CreateOrUpdateAutorestConfigFile() {
     param (
         [string]$autorestFilePath,
+        [string]$namespace,
         [string]$inputfile = "",
-        [string]$readme = ""
+        [string]$readme = "",
+        [string]$autorestConfigYaml = ""
     )
-    if (Test-Path -Path $autorestFilePath) {
-        if ($readme -ne "") {
-            Write-Host "Updating autorest.md file to config required readme file."
+
+    if ((Test-Path -Path $autorestFilePath) -and (![String]::IsNullOrWhiteSpace((Get-Content -Path $autorestFilePath)))) {
+        if (($readme -ne "") -or ($inputfile -ne "")) {
             $requirRex = "require*:*";
```

Comment:
This is not a valid regex. I suspect you want something like `require.*:.*`.

Reply:
Thanks @weshaggard. I updated the regex.